client/driver/docker.go

```diff
@@ -119,13 +107,10 @@ func NewDockerDriverConfig(task *structs.Task) (*DockerDriverConfig, error) {
 	if err := mapstructure.WeakDecode(task.Config, &driverConfig); err != nil {
 		return nil, err
 	}
-	if strings.Contains(driverConfig.ImageName, "https://") {
```

Comment:
@diptanu

I understand your explanation. However, I was confused again, after reading documentation.

https://www.nomadproject.io/docs/drivers/docker.html
~~~
image - The Docker image to run. The image may include a tag
or custom URL and should include
https:// if required. By default it will be fetched from Docker Hub.
~~~~

It says that **should include https:// if required.**. Even if customer includes `https://`, driver will get rid of it soon.
I think your explanation was correct, but the documentation was wrong. Am I correct?